installation/scripts/testing.sh

```diff
@@ -169,7 +177,9 @@ cleanupExitCode=$?
 echo "ClusterTestSuite details:"
 kubectl get cts ${suiteName} -oyaml
 
-deleteCTS ${suiteName}
+if [[ ${CLEANUP} = "true" ]]; then
+    deleteCTS ${suiteName}
```

Comment:
Please use 2 spaces, not 4 :troll:

Reply:
Do you know that there is a mix of indentation in this script? But changed to 2 spaces